src/EnergyPlus/BranchNodeConnections.cc

```diff
@@ -1868,6 +1870,11 @@ namespace BranchNodeConnections {
 				if ( AlreadyNoted( Count ) ) continue;
 				//  All other values must match
 				AlreadyNoted( Other ) = true;
+
+				if ("COILSYSTEM:INTEGRATEDHEATPUMP:AIRSOURCE" == CompSets(Count).CType)	{
```

Comment:
Until we have a better code style guide, new code should be uniform with the existing codebase, so you need to add padding within brackets and parentheses.  This comment should be applied throughout the code changes on this PR.